pkg/core/config/tenant.go

```diff
@@ -105,5 +132,33 @@ func SetTenantConfig(i interface{}, t TenantConfiguration) {
 func NewTenantConfigurationFromEnv(_ *http.Request) (TenantConfiguration, error) {
 	c := TenantConfiguration{}
 	err := envconfig.Process("", &c)
+	c.SSOSetting = appendSSOSetting()
+	c.SSOConfigs = appendSSOConfigs()
+
 	return c, err
 }
+
+func appendSSOSetting() (setting SSOSetting) {
+	envconfig.Process("", &setting)
+	return
+}
+
+func appendSSOConfigs() []SSOConfiguration {
+	configs := make([]SSOConfiguration, 0)
+	SSOProviderNames := []string{
+		"google",
+		"facebook",
+		"instagram",
+		"linkedin",
```

Comment:
Maybe we can put the provider lists in a centralised place? It would be easier for us to add new provide in future.